protocol/scripts/deploys/mainnet/v2/contract-addresses.ts

```diff
@@ -0,0 +1,429 @@
+                FEE_COLLECTOR: '0x4D6175d58C5AceEf30F546C0d5A557efFa53A950',
+                BPT_TOKEN: '0x173063a30e095313eee39411f07e95a8a806014e',
+                POOL: '0x8bd4a1e74a27182d23b98c10fd21d4fbb0ed4ba00002000000000000000004ed',
+                POOL_ID: '79',
+                REWARDS: '0x987c70086239e1a1e9f242ca19fb5a270d4e72b2'
+            },
+      },
+        STRATEGIES: {
+            DSR_BASE_STRATEGY: {
+                ADDRESS: '0xfB12F7170FF298CDed84C793dAb9aBBEcc01E798',
+                EXECUTOR_MSIG: '0x4D6175d58C5AceEf30F546C0d5A557efFa53A950',
+                RESCUER_MSIG: '0x4D6175d58C5AceEf30F546C0d5A557efFa53A950',
+                // No circuit breakers for DSR base strategy
+            },
+            TEMPLE_BASE_STRATEGY: {
+                ADDRESS: '0xc1EeD9232A0A44c2463ACB83698c162966FBc78d',
+                EXECUTOR_MSIG: '0x4D6175d58C5AceEf30F546C0d5A557efFa53A950',
+                RESCUER_MSIG: '0x4D6175d58C5AceEf30F546C0d5A557efFa53A950',
+                // No circuit breakers for Temple base strategy
+            },
+            GNOSIS_SAFE_STRATEGY1: {
+                ADDRESS: '0xbFD3c8A956AFB7a9754C951D03C9aDdA7EC5d638',
+                EXECUTOR_MSIG: '0x4D6175d58C5AceEf30F546C0d5A557efFa53A950',
+                RESCUER_MSIG: '0x4D6175d58C5AceEf30F546C0d5A557efFa53A950',
+                UNDERLYING_GNOSIS_SAFE: '0x67ae2161449cf3c5528fea969beba3e4f3288c61',
+                CIRCUIT_BREAKERS: {
+                    DAI: '0x04f1A5b9BD82a5020C49975ceAd160E98d8B77Af',
+                    TEMPLE: '0xde79380FBd39e08150adAA5C6c9dE3146f53029e',
+                },
+            },
+            RAMOS_STRATEGY: {
+                ADDRESS: '0x96E303b6D807c0824E83f954784e2d6f3614f167',
+                EXECUTOR_MSIG: '0x4D6175d58C5AceEf30F546C0d5A557efFa53A950',
+                RESCUER_MSIG: '0x4D6175d58C5AceEf30F546C0d5A557efFa53A950',
+                CIRCUIT_BREAKERS: {
+                    DAI: '0x87c470437282174b3f8368c7CF1Ac03bcAe57954',
+                    TEMPLE: '0x746a48E39dC57Ff14B872B8979E20efE5E5100B1',
+                },
+            },
+            TLC_STRATEGY: {
+                ADDRESS: '0x64f5219563e28EeBAAd91Ca8D31fa3b36621FD4f',
+                EXECUTOR_MSIG: '0x4D6175d58C5AceEf30F546C0d5A557efFa53A950',
+                RESCUER_MSIG: '0x4D6175d58C5AceEf30F546C0d5A557efFa53A950',
+            },
+        },
+        EXTERNAL: {
+            MAKER_DAO: {
+                DAI_TOKEN: '0x6B175474E89094C44Da98b954EedeAC495271d0F',
+                DAI_JOIN: '0x9759A6Ac90977b93B58547b4A71c78317f391A28',
+                POT: '0x197E90f9FAD81970bA7976f33CbD77088E5D7cf7',
+            },
+            BALANCER: {
+                VAULT: '0xBA12222222228d8Ba445958a75a0704d566BF2C8',
+                HELPERS: '0x5aDDCCa35b7A0D07C74063c48700C8590E87864E',
+                TOKEN: '0xba100000625a3754423978a60c9317c58a424e3D',
```

Comment:
Added this so I could deploy `02-ramos-aura-staking.ts` for mainnet, which I noticed it was used previously as one of the constructor params for current mainnet contract, took the details from `protocol/scripts/deploys/mainnet/ramos/dai/01-ramos-aura-staking.ts`

Reply:
Perhaps call this `BAL_TOKEN` to be explicit - given there's also a Balancer 'BPT' token